doc/source/metadata/Reference.rst

```diff
@@ -0,0 +1,577 @@
+Reference
+=========
+
+.. contents:: Contents
+    :depth: 3
+
+Usage
+-----
+
+This section is intended as a quick look-up on the use of metadata items
+within IMPROVER; they are listed alphabetically. 
+It also indicates whether the metadata item is part of 
+the basic `NetCDF`_  metadata,
+the Climate and Forecast or `CF Metadata Conventions`_
+or specific to IMPROVER
+and whether it **must**, **should** or **could** be present.
+
+Metadata items
+--------------
+
+altitude
+********
+
+This CF coordinate variable holds the height above sea level
+for site data
+and has the ``standard_name`` attribute set to ``altitude``.
+
+This **should** be present for the site data to allow
+the data to be fully exploited.
+
+axis
+****
+
+This CF attribute uses a single capitalised character to indicate
+how a coordinate variable should be intepreted.
+It can take the values ``X``, ``Y``, ``Z`` and ``T``
+representing the three spatial directions and time.
+
+Axis values **should** be set for non-scalar coordinate variables,
+which for IMPROVER gridded data are ``X`` and ``Y``.
+IMPROVER also has ``threshold`` and ``percentile`` 
+coordinate variables, but there are no standards for labelling these.
+
+bounds
+******
+
+This CF attribute provides a label pointing to a separate
+variable defining the bounds of a point on an axis,
+most commonly the start and end of a time period.
+
+This **must** be present if an associated 'bounds variable' exists.
+
+calendar
+********
+
+For IMPROVER, this CF attribute is set to ``gregorian`` to indicate
+that a Gregorian (standard) calendar is used.
+
+cell_methods
+************
+
+This CF attribute can be used to describe the application of simple
+statistical processing to a variable 
+(e.g. a maximum of the temperature over a period of time).
+It is a string comprising of a list of blank-separated words of the form
+``name: method``, where
+the ``name`` can be a dimension of the variable, a scalar coordinate variable,
+a valid standard name, or the word ``area`` and
+the ``method`` should be selected from a standard list 
+described in the `CF Metadata Conventions`_ 
+It is also possible to included additional information
+in the form of a ``comment``
+
+If any method other than ``point`` is specified for a given axis,
+then bounds **must** also be provided for that axis.
+
+For example, ``time: maximum`` would indicate the maximum 
+over the period of the time described by the time bounds.
+
+Cell methods are covered in more detail in the :ref:`stat-section`.
+
+Conventions
+***********
+
+This netCDF attribute specifies a space-separated
+(or, if necessary, comma-separated) 
```

Comment:
```suggestion
(or, if necessary, comma-separated) 
```
Maybe a silly question, but why would a comma-separated list be required?

Reply:
I think this is if the conventions have spaces in their title.